test/integrations/destinations/am/batch/data.ts

```diff
@@ -0,0 +1,3299 @@
+export const data = [
+    {
+        "name": "am",
+        "description": "Test 0",
```

Comment:
Is it possible, just for the error cases (as we can understand the reason of error from the responses), that we add it to the test description, like here

Test 0: ERROR- Sending both userId and deviceId as undefined

Reply:
Yeah we can do that